src/App.tsx

```diff
@@ -1,61 +1,107 @@
+import { useState, ChangeEvent, FormEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todoTitle, setTodoTitle] = useState<string>('');
+  const [todoUser, setTodoUser] = useState<string>('0');
+  const [todos, setTodos] = useState(todosFromServer);
+  const [userError, setUserError] = useState<boolean>(false);
+  const [titleError, setTitleError] = useState<boolean>(false);
+
+  const title = (event: ChangeEvent<HTMLInputElement>) => {
+    setTodoTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const person = (event: ChangeEvent<HTMLSelectElement>) => {
+    setTodoUser(event.target.value);
+    setUserError(false);
+  };
+
+  const createTodo = (e: FormEvent) => {
+    e.preventDefault();
+    const newId
+      = todos.reduce(
+        (acc, curr) => (curr.id > acc.id ? curr : acc), todos[0],
+      ).id + 1;
+
+    if (todoUser === '0') {
+      setUserError(true);
+    }
+
+    if (todoTitle.length === 0) {
+      setTitleError(true);
+    }
```

Comment:
First validate, then calculate things.

Move these if statements to the top of the function. You should also stop the function if one of them is falsy to prevent looking for user when form is invalid.